src/status_im2/navigation/options.cljs

```diff
@@ -4,7 +4,7 @@
 
 (defn default-options
   []
-  {:layout {:orientation :portrait}
+  {:layout {:orientation ["portrait"]}
```

Comment:
actually, we should keep only this one, and orientation prop in other places, and you can keep it as keyword `:portrait`

Reply:
Wasn't the issue that we're using a keyword instead a of vector of strings ?

Reply:
no just keyword instead of vector